### DISCUSSION TITLE
when get to install it, give me blank terminal!

### DISCUSSION THREAD

**Discussion by z3r0xk** [projectdiscovery/mapcidr#192] [Q&A] [answered]
**Describe the bug**
i try to install the tool and nothing happen just give me a blank terminal, and try to do it manually by download it by git command rather than go command and it not work and give me an error 
"go: no module declaration in go.mod. To specify the module path:go mod edit -module=example.com/mod"

**Mapcidr version**
latest 

**Complete command you used to reproduce this**
this is NOT work
go install -v github.com/projectdiscovery/mapcidr/cmd/mapcidr@latest

**Screenshots**
![Screenshot 05-22-2023 10 39 02](https://github.com/projectdiscovery/mapcidr/assets/109469391/20496ecb-9f37-48bc-80e6-ab259ca7f790)

**Comment by ehsandeep** [comment, marked as answer]
@Z3r0xK see https://youtu.be/wtHcXA2l9c4
